[stable/rabbitmq-ha] Add ingress for management and customizable probes

coreypobrien opened the pull request:
Merging features from `stable/rabbitmq` to get toward single rabbit chart for #4748

coreypobrien committed:
[stable/rabbitmq-ha] Add ingress for management and customizable probes

timstoop commented:
/ok-to-test
/lgtm

unguiculus commented on stable/rabbitmq-ha/templates/ingress.yaml:
```diff
@@ -0,0 +1,38 @@
+spec:
+  rules:
+  {{- if .Values.ingress.hostName }}
+  - host: {{ .Values.ingress.hostName }}
```
Make list indentation consistent. The rest of the chart seems to indent two spaces.

unguiculus commented on stable/rabbitmq-ha/templates/ingress.yaml:
```diff
@@ -0,0 +1,38 @@
+  - http:
+  {{- end }}
+      paths:
+        - path: {{ default "/" .path }}
```
Don't use the default function. Defaults should go in to `values.yaml`. Also, it should be `.Values.path`.

unguiculus reviewed (changes requested):
I've made a couple of comments, but I'd suggest you use the ingress template you get with `helm create` using Helm 2.8+.

coreypobrien committed:
Merge remote-tracking branch 'upstream/master' into mergerabbit

coreypobrien committed:
Update indents and defaults for ingress

coreypobrien commented:
@unguiculus I updated the indentation and fixed that path default etc. I'm open to making this whole chart move closer to other standards in general, but I'm trying to keep this minimally different from the `rabbitmq` chart so we can work toward merging them in #4748.

coreypobrien committed:
Merge branch 'master' into mergerabbit

unguiculus commented:
/lgtm
